students/jjakub/session04/trigram.py

```diff
@@ -0,0 +1,50 @@
+
+
+# words = "I wish I may I wish I might".split()
+# wordcnt = 25
+
+# create dictionary: loop method
+# d = {}
+# for i in range(len(words)-2):
+#     pair = tuple(words[i:i + 2])
+#     follower = words[i + 2]  
+#     if pair not in d:
+#         d[pair] = [follower]
+#     else:
+#         d[pair].append(follower)
+
+### create dictionary
+def build_trigrams(words, wordcnt):
+    d = {}
+    for i in range(len(words)-2):
+        pair = tuple(words[i:i + 2])
+        follower = words[i + 2]    
+        d.setdefault(pair,[]).append(follower)
```

Comment:
Minor: Good usage of `.setdefault()`! You may find `collections.defaultdict` more useful in the future... No need to change here.